### PR TITLE
[skip ci] ceph_custom_repo: define apt and rpm key for custom repo

### PR DESCRIPTION
**Pull request by xelexin**
This is backport of commit 4c592066b7c1caaec700af347fc9edf2109c1659

Signed-off-by: Rafal Wadolowki <rwadolowski@cloudferro.com>

### CODE REVIEW

File: roles/ceph-defaults/defaults/main.yml [hunk 1]
```diff
@@ -214,6 +214,7 @@ ceph_iscsi_config_dev: true # special repo for deploying iSCSI gateways
 # a URL to the .repo file to be installed on the targets.  For deb,
 # ceph_custom_repo should be the URL to the repo base.
 #
+ceph_custom_key: https://server.domain.com/ceph-custom-repo-key.asc
```
Comment (guits) [inline roles/ceph-defaults/defaults/main.yml:217]: shouldn't it be commented out by default?

**Comment by guits** [comment]
@xelexin a couple of details:

- you shouldn't push directly in `stable-4.0` branch.
- we usually use `cherry-pick -x`

File: roles/ceph-defaults/defaults/main.yml (hunk 1 above)
Comment (xelexin) [reply roles/ceph-defaults/defaults/main.yml]: good point!

**Comment by guits** [comment]
@xelexin why are you changing the base branch ?

**Review by guits — APPROVED** [review]

**Pull request merged**